go/pkg/client/cas_upload.go

```diff
@@ -280,13 +297,12 @@ func (c *Client) uploadUnified(ctx context.Context, entries ...*uploadinfo.Entry
 				return nil, 0, resp.err
 			}
 			if resp.missing {
```

Comment:
This should be preserved because updateAndNotify will only notify the first client that the blob was missing to avoid double counting. https://github.com/bazelbuild/remote-apis-sdks/blob/master/go/pkg/client/cas_upload.go#L614.

Reply:
So this flag is also used to mark a cache hit/miss. Got it.